dashboard/data/bmg_plate.py

```diff
@@ -257,3 +257,23 @@ def get_activation_inhibition_zscore_dict(
             "outliers": v[1],
         }
     return act_inh_dict
+
+
+def filter_low_quality_plates(
+    df: pd.DataFrame, plate_array: np.ndarray, threshold: float = 0.5
+) -> tuple[pd.DataFrame, np.ndarray]:
+    """
+    Remove plates with Z factor lower than threshold
+    TODO Add alert in dash how many plates were deleted
+
+    :param df: DataFrame with control values
+    :param plate_array: array with plate values
+    :param threshold: Z factor threshold value
+    :return: high quality plates
+    """
+    quality_df = df.copy()
```

Comment:
Any reason for a copy here? it does seem to be modified anywhere in the function

Also in that case it does not have to be returned besides `quality_plates` array

Reply:
I meant that if the `quality_df` is not mutated in any way in the function, there is no real need to return it, you could just return the `quality_plates` without the df as it will just pass the exisitng reference

Then `filter_low_quality_plates` would have signature `(pd.DataFrame, np.ndarray, float) -> np.ndarray`

Reply:
Hmm but wouldn't we like to have a copy with low quality plates also? E.g. in case of comparison plots, report etc.

Reply:
ah nvm sry, now I see that it is masked
In that case its fne